conneg-by-ap/index.html

```diff
@@ -938,26 +963,120 @@ <h3>list profiles</h3>
-          <figcaption>
-            A mapping of the elements within an HTTP <code>Link</code> header communicating alternat representations
-            information to the elements within the Alternate Profiles Data Model.
-          </figcaption>
-        </figure>
+        <section id="http-altp">
+          <h4>HTTP Alternate Representations Data Model Implementation</h4>
+          <p>
+            An HTTP <code>Link</code> is (from [[RFC8288]]) "a typed connection between two resources and is comprised of":
+          </p>
+          <ul>
+            <li>A <em>link context</em>,</li>
+            <li>a <em>link target</em>,</li>
+            <li>a link <em>relation type</em>, and</li>
+            <li>optionally, <em>target attributes</em>.</li>
+          </ul>
+          <p>
+            The <em>link context</em> is the resource for which the Link header is being delivered. The <em>relation type</em>
+            is given with a <code>rel</code> attribute key name, the <em>link target</em> is a URI and other optional
+            <em>target attributes</em> may include <em>profile</em>, given with a <code>profile</code> attribute key name.
+          </p>
+          <p>
+            In <a href="#eg-link-headers-loc"></a>, the first portion of the Link header returned for the request for
+            the <em>link context</em> <code>resource/a</code> has these parts:
+          </p>
+          <ul>
+            <li><em>link target</em> - <code>&lt;http://example.org/resource/a.prof1.ttl&gt;</code></li>
+            <li><em>relation type</em> - <code>rel="self"</code></li>
+            <li><em>target attributes</em> - <code>type="text/turtle"</code> &amp; <code>profile="urn:example:profile:x"</code></li>
+          </ul>
+          <p>
+            Using these <code>Link</code> HTTP header parts, this HTTP Functional Profile is able to communicate
+            alternate profiles for a resource as per the <a href="#altr">Alternate Profiles Data Model</a> with the
+            mapping given in <a href="#fig-altr-link-header-mapping"></a> below.
+          </p>
+          <figure id="fig-altr-link-header-mapping">
+            <table>
+              <tr>
+                <th>Alternate Representations Data Model Element</th><th>Link Header Element</th>
+              </tr>
+              <tr>
+                <td><code>rdf:Resource</code></td><td>the <em>link context</em> URI indicates an instance</td>
+              </tr>
+              <tr>
+                <td><code>altp:Representation</code></td><td>the <em>link target</em> URI indicates an instance</td>
+              </tr>
+              <tr>
+                <td><code>prof:Profile</code></td><td>a <em>target attribute</em> of <code>profile</code> may indicates the URI of an instance</td>
+              </tr>
+              <tr>
+                <td><code>dct:conformsTo</code></td><td>understood to link the <em>link target</em> to the <em>target attribute</em> of <code>profile</code></td>
```

Comment:
Suggestion: Replace "understood to link ... given" with "the implicit relation between the link target and the value of the target attribute profile"

Reply:
Changed as requested